lib/crono_trigger/rollbar.rb

```diff
@@ -21,5 +24,10 @@ def self.handle_exception(ex, record)
       Rollbar.reset_notifier!
       Rollbar::CronoTrigger.handle_exception(ex, record)
     end
+
+    CronoTrigger.config.global_error_handlers << proc do |ex|
+      Rollbar.reset_notifier!
```

Comment:
I think `Rollbar.reset_notifier!` should be called before `#execute` is called, otherwise, the scope set in `#execute` is cleared.

```ruby
def execute
  Rollbar.scoped(...) do
    # do something
  end
end
```

I guess you imitated [Rollbar::Sidekiq](https://github.com/rollbar/rollbar-gem/blob/v2.19.4/lib/rollbar/plugins/sidekiq/plugin.rb#L7-L13), but it clears the scope before processing jobs, not before notifying Rollbar. The threads of Sidekiq process multiple jobs, so they need to clear their scope set by a job before processing another job.

Reply:
If you don't need the rollbar error handler, All you have to do is not requiring `crono_trigger/rollbar`, isn't it?